release.yaml

```
@@ -82,6 +84,28 @@ promoteToPublic:
           fi
   finalize:
     steps:
-      - name: "Register on release registry"
+      - name: "Promote on release registry"
         cmd: |
           echo "pretending to call release registry api"
+
+      # tag is usually in the format `5.3.2`
+      # while version is usually the tag prepended with a v, `v5.3.2`
+      - name: "Slack notification (#announce-engineering)"
+        cmd: |
+          echo "Posting slack notification for release"
+          tag="{{tag}}"
+          changelog_version="${tag}//./}"
```

```suggestion
          changelog_version="${tag//./}"
```
extra `}` slipped in

---

Ah! Good catch.
Thank you.